### ISSUE TITLE
intertrabecular fibrosis

### ISSUE THREAD

**Issue by gawlinski** [obophenotype/human-phenotype-ontology#3623]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
intertrabecular fibrosis.

**Comment by pnrobinson** [comment]
This belongs near this term
https://hpo.jax.org/app/browse/term/HP:0031192
Question to @gawlinski -- are we talking about the right or left ventricle or both?

**Comment by pnrobinson** [comment]
After some review, this is unclear to me -- the term is not commonly used and it could theoretically refer to bone or heart. I am closing this for now, but please reopen @gawlinski with a definition if still needed.
